Engineering-Reference/M365SourceAO.md

```diff
@@ -43,64 +43,59 @@ Default Value: Off
 
 SharePoint Migration API is recommended for all migrations to SharePoint Online and OneDrive for Business. For more information see the article here.
 
-Change Conditions: ??
+Change Conditions: Turn on to use the SharePoint Migration API
```

Comment:
Turn off to use the CSOM API instead of the Migration API. Not recommended unless instructed by support.

Reply:
done